external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/BinaryEventDataSystemPropertiesScheme.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.storm.eventhubs.spout;
+
+import com.microsoft.azure.eventhubs.EventData;
+import org.apache.storm.tuple.Fields;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An Event Data Scheme which deserializes message payload into the raw bytes.
+ *
+ * The resulting tuple would contain two items, the first being the message
+ * bytes, and the second a map of properties that include System Metadata
```

Comment:
>properties [](start = 34, length = 10)

Indicate explicitly that EventData.Properties are not included
or I would actually recommend to go ahead and include eventdata.getProperties() also in the deserialized output.